exporter/clickhousetracesexporter/migrations/000016_dependency_graph_minutes.up.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.dependency_graph_minutes (
+    src LowCardinality(String) CODEC(ZSTD(1)),
+    dest LowCardinality(String) CODEC(ZSTD(1)),
+    duration_quantiles_state AggregateFunction(quantiles(0.5, 0.75, 0.9, 0.95, 0.99), Float64) CODEC(Default),
+    error_count SimpleAggregateFunction(sum, UInt64) CODEC(T64, ZSTD(1)),
+    total_count SimpleAggregateFunction(sum, UInt64) CODEC(T64, ZSTD(1)),
+    timestamp DateTime CODEC(Delta(4), Default)
```

Comment:
Any specific reason why we are using Delta(4)? 
We are using CODEC(DoubleDelta, LZ4) for logs and traces as [this](https://altinity.com/blog/2019/7/new-encodings-to-improve-clickhouse) benchmark by altinity shows that DoubleDelta is more efficient.

Reply:
Updated